guides/gatherers.md

```diff
@@ -312,7 +373,18 @@ facts:
     argument: SBD_DEVICE
 ```
 
-For extra information refer to [trento-project/agent/../gatherers/sbd_test.go](https://github.com/trento-project/agent/blob/main/internal/factsengine/gatherers/sbd_test.go)
+Example output (in Rhai):
+
+```ts
+// sbd_pacemaker
+"yes";
+
+// sbd_startmode
+"always";
+
+// sbd_device
+"/dev/vdc";
```

Comment:
What about adding an extra device for clarity `/dev/vdc;/dev/vdb`? So it cannot be misinterpreted as returning just one device.